hermes.js

```diff
@@ -13,9 +13,8 @@
  */
 
 // IIFE to protect global scope.
+'use strict';
```

Comment:
Probably fine for most needs, but if anyone out there is using this in a script tag, then putting this outside the IIFE might affect other code.